arangod/GeneralServer/SocketTask.h

```diff
@@ -42,8 +42,9 @@ namespace arangodb {
 class ConnectionStatistics;
 
 namespace rest {
-class SocketTask : virtual public IoTask {
```

Comment:
I wonder why I made the IoTask virtual. Maybe just because I had no idea what I was doing. (Same for ListenTask)